app/src/features/surveys/view/SurveyAnimals.tsx

```diff
@@ -70,13 +85,47 @@ const SurveyAnimals: React.FC = () => {
   };
 
   const DeviceFormValues: IAnimalTelemetryDevice = {
-    device_id: 0,
+    device_id: '' as unknown as number,
```

Comment:
Any reason why `0` isn't an acceptable placeholder? Empty string is more likely to cause a runtime error.

Reply:
PO requirement